src/feed-list.scss

```diff
@@ -0,0 +1,117 @@
+@import "./mixins";
+
+$block: #{$fd-namespace}-feed-list;
+$link: #{$fd-namespace}-link;

```

Comment:
Can we take anything referencing `$link` out? We should just have 1 `fd-` parent block per file. You can add addtional `fd-feed-list__link` and attributes you need `fd-feed-list__link--more`, etc.

Reply:
Yea, according to specification the more link should be a default link but with `text-transform: uppercase` So I've added modifier